wpk/run.sh

```diff
@@ -61,12 +166,12 @@ main() {
 
     if [ "${NO_COMPILE}" == false ]; then
       # Execute gmake deps if the version is greater or equal to 3.5
-      if [ $MAJOR -ge 3 ] && [ $MINOR -ge 5 ]; then
+      if [ ${MAJOR} -ge 3 ]; then
```

Comment:
I prefer to keep the old check. This allows building WPK packages from older versions.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/467/commits/1c82879d25119ef9aeb3808e1a4f12747d8edbf0